# coding=utf-8

""" Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
    SPDX-License-Identifier: MIT-0
    Program to gather information from a system
"""

syntax = "proto3";

package mrpc;

service mrpc {
    rpc paraphrase (TextPair) returns (YesNo) {}
}

message TextPair {
    bytes text_a = 1;
    bytes text_b = 2;
}

message YesNo {
    bytes message = 1;
    bytes prediction = 2;
}
